app/views/preferences/basic_preferences/show.html.erb

```diff
@@ -32,5 +32,13 @@
                    :class => "form-select" %>
   </div>
 
+  <div class="mb-3">
+    <%= label_tag "editor_color_scheme", t(".preferred_editor_color_scheme"), :class => "form-label" %>
+    <%= select_tag "editor_color_scheme",
+                   options_for_select(%w[auto light dark].map { |scheme| [t(".editor_color_schemes.#{scheme}"), scheme] },
```

Comment:
The user-facing label "auto" is not clear enough here. There can be confusion as to how three dropdowns relate to each other. How about labelling it as "same as Website" on both `:editor` and `:map`?